### PR TITLE
[SimplifyExpr] Add simplify for dq->arg funcs

### PR DESCRIPTION
**Pull request by mbrookhart**
cc @csullivan @AndrewZhaoLuo

### CODE REVIEW

File: src/relay/transforms/simplify_expr.cc [hunk 1]
```diff
@@ -708,6 +708,44 @@ class SimplifyRSqrt : public DFPatternRewrite {
   DFPattern numerator_;
 };
 
+class SimplifyDQArgFunc : public DFPatternRewrite {
```
Comment (AndrewZhaoLuo) [inline src/relay/transforms/simplify_expr.cc:711]: Nit, add a comment

**Review by AndrewZhaoLuo — APPROVED** [review]
Nice, LGTM

File: src/relay/transforms/simplify_expr.cc [hunk 2]
```diff
@@ -708,6 +708,50 @@ class SimplifyRSqrt : public DFPatternRewrite {
...
+  /*! \brief Pattern input */
+  DFPattern x_;
+  /*! \brief dequantize op */
+  DFPattern dq_;
+  /*! \brief Name of op to simplify */
+  String op_;
+};
+
+class SimplifyDQArgMax : public SimplifyDQArgFunc {
+  /*! Simplify dequantize follwed by argmax */
```
Comment (AndrewZhaoLuo) [inline src/relay/transforms/simplify_expr.cc:738]: Probably want to follow format used in rest of file:

```

/*! \brief Make two consecutive add able to be constant_folded.
 * This pattern matching supports commutative property for addition.
 */
class SimplifyConsecutiveAdd : public DFPatternRewrite {
```

Comment (mbrookhart) [reply src/relay/transforms/simplify_expr.cc]: doh!

**Pull request merged**